### PR TITLE
Updating Nunit to v3

### LINKED ISSUE #351: Update NUnit to version 3
Connects to https://github.com/Particular/PlatformDevelopment/issues/792

This is small enough to fit under maintainer prio.

### PR DESCRIPTION
**Pull request by WojcikMike**
fixes #351

### CODE REVIEW

File: src/ServicePulse.Host.Tests/VerifyAppConstantsJSTextReplacement.cs [hunk 1]
```diff
@@ -19,9 +21,9 @@ public class VerifyAppConstantsJSTextReplacement
         [Test]
         public void app_constants_js_validation()
         {
-            var pathToConfig = Path.Combine("app.constants.js");
+            var pathToConfig = Path.Combine(TestContext.CurrentContext.TestDirectory, "app.constants.js");
```
Comment (WilliamBZA) [inline src/ServicePulse.Host.Tests/VerifyAppConstantsJSTextReplacement.cs]: Why was this change necessary?

Comment (WojcikMike) [inline src/ServicePulse.Host.Tests/VerifyAppConstantsJSTextReplacement.cs]: I added it as after updating to nunit v3 r# test runner changed working directory to: 
C:\Users\Michał\AppData\Local\JetBrains\Installations\ReSharperPlatformVs14_000 which caused this test to fail. Let me verify if it worked with nunit test runner

Comment (WojcikMike) [inline src/ServicePulse.Host.Tests/VerifyAppConstantsJSTextReplacement.cs]: I checked with vs nunit adapter and it uses as a work directory:
C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO 14.0\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\TESTWINDOW
So the test failed. My change make it work for this 2 test runners.

**Comment by WilliamBZA** [comment]
LGTM :+1: 

There is no value in this resulting in a new release of ServicePulse. I vote we assign to the next milestone, merge and wait for a feature update for the next release.
